packages/@angular/cli/tasks/npm-install.ts

```diff
@@ -0,0 +1,61 @@
+import { ModuleNotFoundException, resolve } from '@angular-devkit/core/node';
+
+const Task = require('../ember-cli/lib/models/task');
+import chalk from 'chalk';
+import { spawn } from 'child_process';
+
+
+export default Task.extend({
+  run: async function () {
+    const ui = this.ui;
+    let packageManager = this.packageManager;
+    if (packageManager === 'default') {
+      packageManager = 'npm';
+    }
+
+    ui.writeLine(chalk.green(`Installing packages for tooling via ${packageManager}.`));
+
+    const installArgs = ['install'];
+    if (packageManager === 'npm') {
+      installArgs.push('--quiet');
+    }
+    if (this.packageName) {
+      try {
+        // Verify if we need to install the package (it might already be there).
+        // If it's available and we shouldn't save, simply return. Nothing to be done.
+        resolve(this.packageName, { checkLocal: true, basedir: this.project.root });
+
+        if (!this.save) {
```

Comment:
`save` is not defined. Either add `save` or remove all references to it.

Reply:
`save` _is_ defined as an option for the task.